lib/semigrp.gd

```diff
@@ -174,8 +174,9 @@ DeclareOperation( "SemigroupByGenerators", [ IsCollection ] );
 ##  <Oper Name="AsSemigroup" Arg='C'/>
 ##
 ##  <Description>
-##  If <A>C</A> is a collection whose elements form a semigroup
-##  (see&nbsp;<Ref Filt="IsSemigroup"/>) then <Ref Oper="AsSemigroup"/>
+##  If <A>C</A> is a collection whose elements can be multiplied via
+##  <Ref Oper="\*"/> and thereby forms a semigroup
```

Comment:
Perhaps "thereby" is dangerous here: It is not clear to me whether it shall express an implication (which would be wrong). Why was the old formulation bad?

Reply:
You're right that it is a suspicious formulation. My intention was to make it clear that the elements have to form a semigroup with respect to the `*` operation (as opposed to the `+` operation, say). I will change it to `whose elements form a semigroup under <Ref Oper="\*"/>`.